src/vcpkg/update.cpp

```diff
@@ -54,13 +54,18 @@ namespace vcpkg::Update
         nullptr,
     };
 
+    DECLARE_AND_REGISTER_MESSAGE(UpdateCommandNoManifestMode,
+                                 (),
+                                 "",
+                                 "Error: the update command does not support manifest mode.");
+    DECLARE_AND_REGISTER_MESSAGE(UpdateCommandDidYouMean, (), "", "Did you mean `vcpkg x-update-baseline`?");
+
     void perform_and_exit(const VcpkgCmdArguments& args, const VcpkgPaths& paths)
     {
         if (paths.manifest_mode_enabled())
         {
-            Checks::exit_maybe_upgrade(VCPKG_LINE_INFO,
-                                       "Error: the update command does not currently support manifest mode. Instead, "
-                                       "modify your vcpkg.json and run install.");
+            msg::println(Color::error, msgUpdateCommandNoManifestMode);
+            Checks::msg_exit_maybe_upgrade(VCPKG_LINE_INFO, msgUpdateCommandDidYouMean);
```

Comment:
This would be a good separate PR since there's no reason to touch this file here.